radars/primary/quadrants/open-edx/rings/hold/blips/tech-radar.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Tech Radar",
+    "ring": "Hold",
+    "quadrant": "Open edX",
```

Comment:
I _think_ this may be best under `Techniques`, @davidjoy ?

Reply:
Moved to techniques